src/devices/maverick_et73x.c

```diff
@@ -0,0 +1,253 @@
+//for further information please referre to https://forums.adafruit.com/viewtopic.php?f=8&t=25414
+
+#include "rtl_433.h"
+#include "util.h"
+
+#define MAV_MESSAGE_LENGTH 104
+#define TEMPERATURE_START_POSITION_S1 8
+#define TEMPERATURE_START_POSITION_S2 13
+#define TEMPERATURE_BIT_COUNT 5
+
+static unsigned int msg_converted[26];
+static char msg_hex_combined[26];
+static time_t time_now_t, last_time_t;
+static int32_t session_id;
+
+// quarternary convertion
+int quart(unsigned char param) {
+    int retval = 0;
+
+    if (param=='5')
+        retval = 0;
+    if (param=='6')
+        retval = 1;
+    if (param=='9')
+        retval = 2;
+    if (param=='a' || param=='A')
+        retval = 3;
+
+    return retval;
+}
+
```

Comment:
Add a table where the indexes are the values you need.

quart_tab[10] = {0,0,0,0,0,1,0,0,2,3} should do it.